model-engine/model_engine_server/inference/sync_inference/start_fastapi_server.py

```diff
@@ -10,7 +10,7 @@
 
 def start_server():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--graceful-timeout", type=int, default=600)
+    parser.add_argument("--graceful-timeout", type=int, default=1800)
```

Comment:
is this what's being used in async tasks?

Reply:
i thought we are going to patch llm-engine/model-engine/model_engine_server/inference/async_inference/celery.py to listen to sigterm?

Reply:
yeah iirc this code is used to serve the user container for both sync and async tasks for artifact-like bundles, and `async_inference/celery.py` shouldn't be used anymore

Reply:
we'd also have to patch celery-forwarder to listen to sigterm

Reply:
possible to simulate such an scenario by sending traffic while restarting a pod?